modules/gitops-verifying-argocd-agent-installation-in-managed-or-autonomous-mode.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assembly:
+//
+// * argocd-agent-installation/gitops-verifying-argocd-agent-installation-in-managed-or-autonomous-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-verifying-argocd-agent-installation-in-managed-or-autonomous-mode_{context}"]
+= Verifying the Argo CD Agent installation in Managed Mode or Autonomous Mode
+
+Use this procedure to verify that the Argo CD Agent is correctly installed and connected to the Principal cluster in managed or autonomous mode.
+
+.Procedure
+
+. Create an `AppProject` in the agent namespace.
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: default
+  namespace: <AGENT_NAMESPACE>
+spec:
+  clusterResourceWhitelist:
+  - group: '*'
+    kind: '*'
+  destinations:
+  - name: '*'
+    namespace: '*'
+    server: '*'
+  sourceRepos:
+  - '*'
+----
++
```

Comment:
This step, creating an AppProject, is a *setup* step for autonomous mode, not a *verification* step.

This step should be move to autonomous mode install section, as in the gist: https://gist.github.com/jgwest/da80a952732ca026e21c156cca6ab5fe